sbws/lib/relaylist.py

```diff
@@ -11,6 +13,18 @@
 log = logging.getLogger(__name__)
 
 
+class RelayNS(RouterStatusEntryV3):
```

Comment:
I think instead of inheriting RouterStatusEntryV3, we should

- not inherit anything
- rename to Relay (since we're going to pull info from more than the network status consensus)
- init with just a fingerprint
- use the fingerprint to ask stem for relay information (like with `get_network_status` and/or `get_server_descriptor` and/or `get_microdescriptor`) store what we get
- define `@property`s for all the things we need like exit_policy and fingerprint and nickname and flags and its ed25519 certificate and ...

Reply:
sounds good, maybe we should rename some "relay" vars that won't be Relay objs after these changes (not sure if/which now)